### PR TITLE
pre-commit

### PR DESCRIPTION
**Pull request by eggplants**
To make easier format and organize whole codes, I added `pre-commit` and fixed detected errors.

You can see the `pre-commit` hooks:
https://github.com/lxgr-linux/pokete/pull/175/files#diff-63a9c44a44acf85fea213a857769990937107cf072831e1a26808cfde9d096b9

To use `pre-commit`:

```
pip install pre-commit
pre-commit install
pre-commit run --all-files
```

### CODE REVIEW

File: DevGuide.md [hunk 1]
```diff
@@ -1,12 +1,14 @@
...
+
 This guide only handles adding new types, attacks and Poketes to Pokete, for further information on class and function syntax see the html documentation or `$ pydoc <file>`.
 
 1. [Adding Poketes](#adding-poketes)
-2. [Adding attacks](#adding-attacks)
-3. [Adding types](#adding-types)
-4. [Adding subtypes](#adding-subtypes)
+1. [Adding attacks](#adding-attacks)
+1. [Adding types](#adding-types)
+1. [Adding subtypes](#adding-subtypes)
```
Comment (lxgr-linux) [inline DevGuide.md:8]: Making pre-commit change markdown files completely kills the purpose of markdown, markdown should be readable by mashines and humans and for example replacing all numbers in a list with one dues not help that.
Also why doe it escape `[]`?

**Review by lxgr-linux — CHANGES_REQUESTED** [review]

File: DevGuide.md (hunk 1 above)
Comment (eggplants) [reply DevGuide.md]: I adopt `mdformat` for formatting markdown files, but it seems to have some errors. I replaced it with `markdownlint-fix`.

File: Pipfile [hunk 2]
```diff
@@ -5,12 +5,15 @@ name = "pypi"
 
 [packages]
 scrap-engine = ">=1.2.0"
-pynput = {version = ">=1.7.6", sys_platform = "!= 'linux'"}
-playsound = "*"
-pygobject = "*"
+pynput = {version = ">=2.11", sys_platform = "!= 'linux'"}
```
[the remark this answers is not in the document]
Comment (eggplants) [reply Pipfile]: I was based on bit old master branch. Fixed: ae16df3609108e3964d82c657659f1eec10bae8b

[the remark this answers is not in the document]
Comment (eggplants) [reply Pipfile]: I was based on bit old master branch. Fixed: 2461901d4c266671bc69f2d7cbf5f1dfb90224ef

File: DevGuide.md (hunk 1 above)
Comment (eggplants) [reply DevGuide.md]: I adopt `mdformat` for formatting markdown files, but it seems to do some problematic behavior. I dropped and replaced it with `markdownlint-fix`.

**Comment by eggplants** [comment]
Is there any chance to merge? If so please reject this PR.

**Comment by lxgr-linux** [comment]
I talked to some other people and came to the conclusion, that pre-commit may not be very usefull, because it's not mandatory and may result in a lot of trouble.

**Pull request closed without merging**